backend/.gitignore

```diff
@@ -21,3 +21,5 @@ htmlcov/
 ~$*
 .DS_Store
 venv
+
+docs
```

Comment:
Not sure we want that ... I was hoping to store the version current docs in the repo instead of having to force someone to install the repo in order to generate the docs.

Reply:
wouldn't that double the size of our repo?